oss-cloud-backend/serverless.yml

```diff
@@ -5,6 +5,7 @@ provider:
   runtime: nodejs10.x
 
 plugins:
+  - serverless-dotenv-plugin # used to store github access token locally to keep it secret
```

Comment:
We can provide env variables when running the Serverless application offline. Simply start the app with
`ENV=value sls offline`. Let's discuss storing sensitive information separately.